dd-sdk-android/src/test/kotlin/com/datadog/android/DatadogTest.kt

```diff
@@ -103,9 +103,9 @@ internal class DatadogTest {
 
     @Test
     fun `fails if stop called without initialize`() {
-        assertThrows<IllegalStateException> {
+        assertThatThrownBy {
```

Comment:
Why use this one instead of the other ? Don't they do the same thing ?

Reply:
it was an error on AS saying that the previous method requires JAVA 1.8. You don't have it on your end ?